tools/docker/builder/openwrt/scripts/build-openwrt.sh

```diff
@@ -12,8 +12,23 @@ mkdir -p files/etc
 printf '%s=%s\n' "OPENWRT_REPOSITORY" "$OPENWRT_REPOSITORY" >> files/etc/prplwrt-version
 printf '%s=%s\n' "OPENWRT_VERSION" "$OPENWRT_VERSION" >> files/etc/prplwrt-version
 if [ "$TARGET_PROFILE" = DEVICE_NETGEAR_RAX40 ] ; then
-    ./scripts/gen_config.py intel_mips
-    cat profiles/intel_mips.yml >> files/etc/prplwrt-version
+    # Add prplmesh to the list of packages of the profile:
+    sed -i 's/packages:/packages:\n  - prplmesh-dwpal/g' profiles/intel_mips.yml
+    yq write --inplace profiles/netgear_rax40.yml feeds -f profiles_feeds/netgear-rax40.yml
+    ./scripts/gen_config.py netgear_rax40 debug
+    cat profiles_feeds/netgear-rax40.yml >> files/etc/prplwrt-version
+elif [ "$TARGET_PROFILE" = DEVICE_AX6000_2000_ETH_11AXUCI ]; then
+    # Add prplmesh to the list of packages of the profile:
+    sed -i 's/packages:/packages:\n  - prplmesh-dwpal/g' profiles/axepoint.yml
+    yq write --inplace profiles/axepoint.yml feeds -f profiles_feeds/netgear-rax40.yml
+    ./scripts/gen_config.py axepoint debug
+    cat profiles_feeds/netgear-rax40.yml >> files/etc/prplwrt-version
+elif [ "$TARGET_PROFILE" = INTEL_MIPS ]; then
+    # Add prplmesh to the list of packages of the profile:
+    sed -i 's/packages:/packages:\n  - prplmesh-dwpal/g' profiles/intel_mips.yml
+    yq write --inplace profiles/intel_mips.yml feeds -f profiles_feeds/netgear-rax40.yml
+    ./scripts/gen_config.py intel_mips debug
+    cat profiles_feeds/netgear-rax40.yml >> files/etc/prplwrt-version
```

Comment:
At the moment the only difference between the 3 if blocks is `netgear-rax40.yml`.
If we could have "axepoint.yml" and "intel_mips.yml" (if their content is the same, maybe using symlinks) it could be refactored into a single case block.
We would need to path "TARGET_DEVICE" to the Dockerfile for it to work though.

So I mean something like this:
```
case $TARGET_DEVICE in 
netgear-rax40|intel_mips|axepoint)
     sed -i 's/packages:/packages:\n  - prplmesh-dwpal/g' profiles/${TARGET_DEVICE}.yml
     ./scripts/gen_config.py $TARGET_DEVICE debug
     cat profiles_feeds/${TARGET_DEVICE}.yml >> files/etc/prplwrt-version
*)
<nl80211...>
```


(nitpick)

Reply:
I'll just wait to see that CI passes as is, then apply your (good) suggestion